lib/active_interactor/organizer/interactor_interface.rb

```diff
@@ -87,12 +87,13 @@ def execute_inplace_callback(target, callback)
         resolve_option(target, callbacks[callback])
       end
 
+      # Executes after_perform callbacks that have been deferred on the interactor
       def execute_deferred_after_perform_callbacks(context)
         return unless deferred_after_perform_callbacks.present?
 
         interactor = interactor_class.new(context)
         env = ActiveSupport::Callbacks::Filters::Environment.new(interactor, false, nil)
-        deferred_after_perform_callbacks.compile.invoke_after(env)
+        deferred_after_perform_callbacks.compile(nil).invoke_after(env)
```

Comment:
@aaronmallen Adding this so that the Ruby 2.7 check passes seems to be what's causing [my issue](https://github.com/aaronmallen/activeinteractor/pull/371#issuecomment-1836666628). Given that this never included `(nil)` to begin with, I'm wondering if it's okay to remove this? Just wondering on the best way to proceed given the failing checks on the PR. 😅  Any ideas?